v22.1/recommended-production-settings.md

```diff
@@ -312,6 +312,14 @@ For guidance on sizing, validating, and using connection pools with CockroachDB,
 
 {% include {{ page.version.version }}/prod-deployment/monitor-cluster.md %}
 
+## Backup and restore
+
+CockroachDB is built to be fault-tolerant and to recover automatically, but sometimes disasters happen. Having a [disaster recovery](disaster-recovery.html) plan enables you to recover quickly, while limiting the consequences.
+
+Taking regular backups of your data in production is an operational best practice. You can create [full](take-full-and-incremental-backups.html#full-backups) or [incremental](take-full-and-incremental-backups.html#incremental-backups) backups of a cluster, database, or table. We recommend taking backups to [cloud storage](use-cloud-storage-for-bulk-operations.html) and enabling object locking to protect the validity of your backups. CockroachDB supports Amazon S3, Azure Storage, and Google Cloud Storage for backups.
```

Comment:
nit: should we link to https://www.cockroachlabs.com/docs/stable/use-cloud-storage-for-bulk-operations.html#object-locking when talking about object locking?